commit f4b935acdd822727ea1f95219d58feb46a6dd79d
Author: nekaru <32153003+ninajika@users.noreply.github.com>
Date:   Mon Jan 6 14:44:52 2025 +0700

    fix issue https://github.com/Perfare/UnityLive2DExtractor/issues/25

diff --git a/UnityLive2DExtractor/Program.cs b/UnityLive2DExtractor/Program.cs
index ab9dd15..19b05f8 100644
--- a/UnityLive2DExtractor/Program.cs
+++ b/UnityLive2DExtractor/Program.cs
@@ -327,7 +327,7 @@ namespace UnityLive2DExtractor
         {
             var reader = physics.reader;
             reader.Reset();
-            reader.Position += 28; //PPtr<GameObject> m_GameObject, m_Enabled, PPtr<MonoScript>
+            reader.Position += 30; //PPtr<GameObject> m_GameObject, m_Enabled, PPtr<MonoScript>
             reader.ReadAlignedString(); //m_Name
             var cubismPhysicsRig = new CubismPhysicsRig(reader);
 
